src/Agent/NewRelic/Agent/Core/NewRelic.Agent.Core.Commands/CommandService.cs

```diff
@@ -20,12 +21,15 @@ public class CommandService : DisposableService
 
         private readonly IScheduler _scheduler;
 
-        public CommandService(IDataTransportService dataTransportService, IScheduler scheduler)
+        private readonly IConfigurationService _configurationService;
+
+        public CommandService(IDataTransportService dataTransportService, IScheduler scheduler, IConfigurationService configurationService)
         {
             _dataTransportService = dataTransportService;
+            _configurationService = configurationService;
             _scheduler = scheduler;
 
-            _scheduler.ExecuteEvery(GetAndExecuteAgentCommands, TimeSpan.FromMinutes(1));
+            _scheduler.ExecuteEvery(GetAndExecuteAgentCommands, _configurationService.Configuration.GetAgentCommandsCycle ?? TimeSpan.FromMinutes(1));
```

Comment:
Should we move the default value into DefaultConfiguration so that the logic for the harvest cycle is in just one location?